components/details.jsx

```diff
@@ -11,6 +12,20 @@ class Details extends React.Component {
     ReactGA.event(this.props.createGaEventConfig(`Get involved`, `Clicked`, `beacon`));
   }
 
+
+  renderTimePosted() {
+    if (!this.props.created && !this.props.publishedBy) return null;
```

Comment:
do we want to preserve the element spacing by not returning null but simply not setting a time inside the `<p>...</p>`?